src/pages/RFOX/hooks/useGetUnstakingRequestQuery.ts

```diff
@@ -66,21 +72,29 @@ export const useGetUnstakingRequestQuery = <SelectData = UnstakingRequest>({
     [contracts],
   )
 
-  const getUnstakingRequestQueryFn = useMemo(
-    () =>
-      unstakingRequestCountResponse && unstakingRequestCountResponse?.valueOf() > 0n
-        ? () =>
-            multicall(client, {
-              contracts,
-            }).then(r => r.map(response => response.result).filter(isSome))
-        : skipToken,
-    [contracts, unstakingRequestCountResponse],
-  )
+  const getUnstakingRequestQueryFn = useMemo(() => {
+    if (isUnstakingRequestCountLoading || isUnstakingRequestCountPending) return skipToken
+    if (isUnstakingRequestCountError) return () => Promise.reject(unstakingRequestCountError)
+    if (unstakingRequestCountResponse === 0n) return () => Promise.resolve([])
```

Comment:
We have a success `unstakingRequestCountResponse`, but it's a 0 unstaking requests count. We don't need to fire an XHR, and can simply make this query succesful, with an empty array.

Reply:
The planet will thank you ser